watson/cli.py

```diff
@@ -322,8 +322,10 @@ def status(watson):
               help="Reports activity only for frames containing the given "
               "tag. You can add several tags by using this option multiple "
               "times")
+@click.option('-j', '--json', 'json_output', is_flag=True,
+              help="Outputs the report as JSON instead of writing to the console")
```

Comment:
Again, output still goes to console (stdout), only format changes.
 
Also this line's length is currently >79 and make the flake8 test fail.